frontend/src/components/FormComponents/FormButton.jsx

```diff
@@ -23,7 +23,12 @@ class FormButton extends Component {
   render() {
     const { onClick, title } = this.props;
     return (
-      <button type="button" className={this.getClass()} onClick={onClick}>
+      <button
+        type="button"
+        className={this.getClass()}
+        onClick={onClick}
+        style={this.props.style}
```

Comment:
Nitpick - would you mind accessing the `style` prop on line 24, like `onClick` and `title`, instead of here?